src/azure-cli/azure/cli/__main__.py

```diff
@@ -47,27 +50,24 @@ def cli_main(cli, args):
     else:
         telemetry.set_success()
 
-    elapsed_time = timeit.default_timer() - start_time
-
     sys.exit(exit_code)
 
 except KeyboardInterrupt:
     telemetry.set_user_fault('keyboard interrupt')
     sys.exit(1)
 except SystemExit as ex:  # some code directly call sys.exit, this is to make sure command metadata is logged
     exit_code = ex.code if ex.code is not None else 1
-
-    try:
-        elapsed_time = timeit.default_timer() - start_time
-    except NameError:
-        pass
-
     raise ex
 
 finally:
     telemetry.conclude()
 
     try:
-        logger.info("command ran in %.3f seconds.", elapsed_time)
+        # Log the invoke finish time
+        invoke_finish_time = timeit.default_timer()
+        logger.info("Command ran in %.3f seconds (init: %.3f, invoke: %.3f)",
```

Comment:
I am not sure if this **running time info** should be moved to `logger.debug`. I am open to this.